RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml

```diff
@@ -14,47 +15,51 @@
 </div>
 
 
-<form action="" method="GET">
+<form action="" method="GET" >
 
-    First name: <input type="text" name="search">
+    First name: <input type="text" name="search" value=@search> @*Morgan added "value" tag to repopulate the field with the last searched string value*@

```

Comment:
Added the value tag to your form input box. By setting it to the new variable 'search' I added at the top it will pull in the last searched string to the input field. This way your users will know what they searched last.

Reply:
The value of the commentary here in the pull request is high... but in code, it's clutter.